web/src/HUD.tsx

```diff
@@ -60,10 +60,6 @@ export default class HUD extends Component<HudProps, HudState> {
     this.unlisten = this.history.listen((location: any, action: string) => {
       let tags = navigationToTags(location, action)
       incr("ui.web.navigation", tags)
-
-      this.handleClearHighlight()
-      let selection = document.getSelection()
-      selection && selection.removeAllRanges()
```

Comment:
@lizzthabet this was the line that was causing the problem :hot_face:

Reply:
ahhh! the culprit